master/buildbot/data/builds.py

```diff
@@ -53,7 +54,13 @@ def get(self, resultSpec, kwargs):
             bldr = kwargs['builderid']
             num = kwargs['number']
             dbdict = yield self.master.db.builds.getBuildByNumber(bldr, num)
-        defer.returnValue((yield self.db2data(dbdict))
+
+        try:
+            props = yield self.master.db.builds.getBuildProperties(dbdict['id'])
```

Comment:
This should be done only when there is a filter requesting explicitly properties.

data.get(("builds"), FilterSpec(property=["slavename", "owner"])

which translate on rest to
/api/v2/builds?property=slavename&property=owner&limit=10